math/fibonacci_fast.cpp

```diff
@@ -39,19 +44,120 @@ uint64_t fib(uint64_t n) {
             << "Cannot compute for n>93 due to limit of 64-bit integers\n";
         return 0;
```

Comment:
instead of output to cerr, throw this as an invalid_argument error instead.

Reply:
I have already revised it to the following content:

    if (n >= MAX) {
        throw std::invalid_argument("Cannot compute for n>=" + std::to_string(MAX) +
                                    " due to limit of 64-bit integers");
        return 0;
    }